src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/DataSetVersionMappingControllerTests.cs

```diff
@@ -1193,17 +1455,36 @@ public async Task UpdateMappingDoesNotExist_Returns400_AndRollsBackTransaction()
                                 }
                             }
                         }
+                    },
+                    Candidates =
+                    {
+                        {
+                            "Target filter 1 key",
+                            new FilterMappingCandidate("Filter 1")
+                            {
+                                Options =
+                                {
+                                    {
+                                        "Target filter 1 option 1 key", new MappableFilterOption("Filter 1 option 1 label")
+                                    },
+                                    {
+                                        "Target filter 1 option 2 key", new MappableFilterOption("Filter 1 option 2 label")
+                                    },
+                                    { "Target filter 1 option 3 key", new MappableFilterOption("Filter 1 option 3 label") }
```

Comment:
might be nice to format this the same as the ones above :)

Reply:
I've given this file a blanket reformat, which cleans this one up too!